WooCommerce/Classes/System/FeatureFlag.swift

```diff
@@ -4,10 +4,14 @@ enum FeatureFlag: Int {
     /// Throwaway case, to prevent a compiler error:
     /// `An enum with no cases cannot declare a raw type`
     case null
+    case manualTracking
```

Comment:
Just a quick question → what will this flag be used for? (I didn't see it referenced in this PR. 😄)

Reply:
@bummytime Oops, I forgot to mention that. I added it in case it is necessary to hide the UI by the next code freeze. I will remove it again for now, and add it later on, only if necessary.